src/styles/blocks/card.scss

```diff
@@ -0,0 +1,125 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  max-width: $card-width;
+  padding: 32px 16px 0;
+  margin-right: $card-right-indent;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-top: 46px;
+  }
+
+  &__logo {
+    margin: 0 auto;
+
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    margin-top: 40px;
+
+    font-weight: 500;
+    font-size: 12px;
+
+    color: #060b35;
+
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 4px;
+    margin-bottom: 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+
+    color: #616070;
+
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    margin-bottom: 24px;
+
+    color: #060b35;
+
+    line-height: 18px;
+  }
+
+  &__review {
+    font-size: 10px;
+    font-weight: 400;
+
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    font-weight: 400;
+
+    color: #616070;
+
+    line-height: 18px;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+
+    color: #060b35;
+
+    line-height: 18px;
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 166px;
+    height: 40px;
+    padding: 12px;
+    margin-bottom: 16px;
+
+    background-color: #00acdc;
+    color: #fff;
+
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    text-decoration: none;
+    text-transform: uppercase;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  &__buy-button:hover {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 10th point